src/helpers/createNotificationHelper.ts

```diff
@@ -0,0 +1,23 @@
+import PushNotification from 'react-native-push-notification';
+import { AuthReducerState } from '../redux/reducers/authReducer';
+import { TodoItemType } from '../models';
+
+export type CreateNotificationHelperPropsType = {
+  channelId: AuthReducerState['deviceToken'];
+  newItem: TodoItemType;
+};
+
+export const createNotificationHelper = (
+  props: CreateNotificationHelperPropsType,
```

Comment:
props it is just for components, please rename that argument

Reply:
i rename this